irispy/data/_sample.py

```diff
@@ -0,0 +1,97 @@
+# -*- coding: utf-8 -*-
+"""IrisPy Sample and test files"""
+
+import os.path
+import socket
+import warnings
+from zipfile import ZipFile
+from shutil import move
+
+from astropy.utils.data import download_file
+
+from sunpy.util.net import url_exists
+from sunpy.util.config import get_and_create_sample_dir
+from sunpy import config
+
+__author__ = "Steven Christe"
+__email__ = "steven.christe@nasa.gov"
+
+# Creating the directory for sample files to be downloaded
+sampledata_dir = get_and_create_sample_dir()
+
+# urls to search for the data
+_BASE_URLS = (
+    'https://github.com/sunpy/sample-data/blob/master/irispy/,
+    'http://data.sunpy.org/sample-data/',
+    'http://hesperia.gsfc.nasa.gov/~schriste/sunpy-sample-data/'
+    )
+
+_FILES = {"SJI_CUBE_2832": ("iris_l2_20170502_052551_3893010094_SJI_2832_t000.fits", ""),
+          "SJI_CUBE_1330": ("iris_l2_20170502_052551_3893010094_SJI_1330_t000.fits", ""),
+          "SJI_CUBE_2796": ("iris_l2_20170502_052551_3893010094_SJI_2796_t000.fits", ""),
+          "SJI_CUBE_1400": ("iris_l2_20170502_052551_3893010094_SJI_1400_t000.fits", ""),
+          "RASTER": ("iris_l2_20170502_052551_3893010094_raster.fits", ".tar.zip")
+          }
+
+sample_files = {}
+for key in _FILES:
+    sample_files[key] = os.path.abspath(os.path.join(sampledata_dir, _FILES[key][0]))
+
+
+def download_data(progress=True, overwrite=True, timeout=None):
```

Comment:
Should we not then call this ```download_sample_data``` to be consistent?  Or is there a reason you've named it differently?